### PR TITLE
ExtraHop - Add incorrectly deprecated playbook back into ExtraHop Pack

### PR DESCRIPTION
**Pull request by Dan-at-Extrahop**
## XSOAR [Partner](https://xsoar.pan.dev/docs/partners/become-a-tech-partner) - ExtraHop
In case you are a technological partner:
- [x] The title must be in the following format: **[YOUR_PARTNER_ID] short description**.

If you are not a technological partner (i.e. PANW employee, customer or individual developer), please delete this section.

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://github.com/demisto/content/pull/7326

## Description
In the process of converting the ExtraHop integration to a content pack, one of our required playbooks was incorrectly deprecated.

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by Dan-at-Extrahop** [comment]
FYI @mchasepan, I wanted to get a fix in asap.

**Comment by mchasepan** [comment]
Thanks, Dan. Was just reviewing w @ShahafBenYakir too.

**Comment by Dan-at-Extrahop** [comment]
> Thanks, Dan. Was just reviewing w @ShahafBenYakir too.

Great thank you. I also just added the pack README updates we discussed today here. I'm unsure of the CI validation errors, please feel free to make the necessary changes or advise.

**Comment by ShahafBenYakir** [comment]
Hi @Dan-at-Extrahop as the validation suggests please run `demisto-sdk update-release-notes -i Packs/ExtraHop -u revision`. If you don't have the demisto-SDK on your local machine [see this](https://github.com/demisto/demisto-sdk#installation) on how to download it.

**Comment by Dan-at-Extrahop** [comment]
@ShahafBenYakir, my organization policies do not allow me to accept the CircleCI terms to view all of my public and private repos, so I can not see this validation output. In the interest of time in getting this issue resolved, could you please make the necessary updates to this PR? Thank you

File: Packs/ExtraHop/ReleaseNotes/1_0_3.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Playbooks
+##### ExtraHop - Ticket Tracking
+- Re-enabled the playbook as it was deprecated.
```
Comment (ShahafBenYakir) [inline Packs/ExtraHop/ReleaseNotes/1_0_3.md:4]:
```suggestion
- Playbook was un-deprecated and is now available.
```

**Review by ShahafBenYakir — APPROVED** [review]

**Comment by Dan-at-Extrahop** [comment]
Thank you @ShahafBenYakir

**Comment by ShahafBenYakir** [comment]
@Dan-at-Extrahop Unfortunately due to these changes we will have to release this playbook as v2. It will not impact existing users (users that have already downloaded this playbook) but new users will see it as v2.

**Comment by Dan-at-Extrahop** [comment]
> @Dan-at-Extrahop Unfortunately due to these changes we will have to release this playbook as v2. It will not impact existing users (users that have already downloaded this playbook) but new users will see it as v2.

Ok, understood. Thank you.

**Comment by Dan-at-Extrahop** [comment]
> @Dan-at-Extrahop Unfortunately due to these changes we will have to release this playbook as v2. It will not impact existing users (users that have already downloaded this playbook) but new users will see it as v2.

@ShahafBenYakir I believe this change to v2 means that [ExtraHop - Default](https://github.com/demisto/content/blob/master/Packs/ExtraHop/Playbooks/playbook-ExtraHop_-_Default.yml#L114) playbook will now need to point to ExtraHop - Ticket Tracking v2?  I believe the `name` and `playbook` fields need to be updated, but I'm unsure if that's the extent of the association. 

I was alerted by an SE that ticket tracking appeared to not be working on a fresh install from the marketplace. Thanks

**Pull request merged**